Makefile

```diff
@@ -9,6 +9,8 @@ PWD := $(shell pwd)
 
 MAKEFLAGS += -B
 
+SLAVE_IMAGE=sonic-slave-$(USER)-$(shell shasum sonic-slave/Dockerfile | awk '{print substr($$1,0,11);}')
```

Comment:
instead of embed the shasum into the docker image name, can we keep the name as it is and use tag? In this case, we can use a latest tag and know which docker-slave is used. otherwise, the dev box may have lots of docker images and we do not know which one is the latest easily.